subs/pantry/src/Pantry/Types.hs

```diff
@@ -890,6 +926,11 @@ instance PersistFieldSql SafeFilePath where
 unSafeFilePath :: SafeFilePath -> Text
 unSafeFilePath (SafeFilePath t) = t
 
+safeFilePathtoPath :: Path Abs Dir -> SafeFilePath -> IO (Path Abs File)
```

Comment:
This could probably be a `MonadThrow` constraint, right?

Reply:
Done.